src/marqo/README.md

```diff
@@ -4,47 +4,88 @@ you want.
 
 See [here](https://github.com/marqo-ai/marqo/blob/mainline/CONTRIBUTING.md#unit-tests) for how to run unit tests.
 
-## Select an option (from A-E) to get set up. In most cases, Option A is recommended. 
+## Running Marqo locally (outside of docker) for development
 
-### Option A. Run the Marqo application locally (outside of docker):
+There are two ways to run Marqo locally (outside of docker) for development: Option A. through `uvicorn`, 
+Option B. through your IDE (e.g. PyCharm). 
+We highly recommend using the Option A, as it allows you to set breakpoints and debug Marqo. 
+Before running Marqo locally, you will need to do some preparations to set up Vespa locally.
 
-1. Have a running Marqo-OS instance available to use. You can spin up a local instance with the following command
-(if you are using an arm64 machine, replace `marqoai/marqo-os:0.0.3` with `marqoai/marqo-os:0.0.3-arm`):
+### Preparations
+
+1. Clone the Marqo Github repo and cd into it
 ```bash
-docker run --name marqo-os -id -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3
+git clone https://github.com/marqo-ai/marqo.git
+cd marqo
 ```
 
-2. Clone the github repo
+2. Install Marqo dependencies
+```bash
+pip install -r requirements.txt
```

Comment:
just curious, isn't it a place where we need to recommend requirements.dev.txt?

Reply:
Thanks for getting this. Let me update it